tests/test_kalman_filter.py

```diff
@@ -0,0 +1,167 @@
+"""Unit tests for Kalman filter."""
+
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.algebra_utils import from_vector_to_diagonal_matrix
+from geomstats.geometry.matrices import Matrices
+from geomstats.learning.kalman_filter import KalmanFilter
+from geomstats.learning.kalman_filter import Localization
+from geomstats.learning.kalman_filter import LocalizationLinear
+
+
+class TestKalmanFilter(geomstats.tests.TestCase):
+    _multiprocess_can_split_ = True
+
+    def setUp(self):
+        gs.random.seed(123)
+        self.linear_model = LocalizationLinear()
+        self.nonlinear_model = Localization()
+        self.kalman = KalmanFilter(self.linear_model)
+        self.prior_cov = gs.eye(2)
+        self.process_cov = gs.eye(1)
+        self.obs_cov = 2. * gs.eye(1)
+
+    def test_LocalizationLinear(self):
+        initial_state = gs.array([0.5, 1.])
+        time_step = 0.5
+        acc = 2.
+        increment = gs.array([time_step, acc])
+        measurement = gs.array([0.7])
+
+        # Test propagation
```

Comment:
These different tests should be in different `test_*` functions, in order to have "unit" tests. This way, when travis fails, we directly know which part of the code is failing: using the name `test_*`.

Would it be possible to make these tests unit?